NEWS.md

```diff
@@ -1,5 +1,8 @@
 # pkgdown (development version)
 
+* All third-party resources have been updated to the latest versions for 
+  Bootstrap 5 (@IndrajeetPatil, #2153).
```

Comment:
I am not sure if the same should also be done for [BS3](https://github.com/r-lib/pkgdown/blob/main/inst/BS3/templates/head.html).